src/entities/session.entity.ts

```diff
@@ -24,4 +25,7 @@ export class SessionEntity extends BaseEntity {
   @ManyToOne(() => CourseEntity, (courseEntity) => courseEntity.session)
   @JoinTable({ name: 'course', joinColumn: { name: 'courseId' } })
   course: CourseEntity;
+
+  @ManyToMany(() => SessionUserEntity, (sessionUserEntity) => sessionUserEntity.session)
```

Comment:
i think this is a ManyToOne Session -> SessionUser

As the SessionUser record will only point to 1 session. A new SessionUser record will be created for each session the user interacts with